src/onegov/org/views/newsletter.py

```diff
@@ -80,13 +80,62 @@ def get_newsletter_form(
     return form
 
 
-def news_by_newsletter(
+def newsletter_news_by_access(
     newsletter: Newsletter,
-    request: 'OrgRequest'
+    request: 'OrgRequest',
+    access: str = 'public',
+    include_wider_access: bool = True,
```

Comment:
I don't think we ever want to exclude wider access levels. So until we have a use-case for it I would get rid of this argument and the corresponding if/else below.

Reply:
Absolutely!